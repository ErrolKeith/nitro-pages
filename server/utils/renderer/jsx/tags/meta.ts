import { makeJsx } from "./makeJsx";

export interface MetaProps {
  name: string;
  content: string;
}

export function makeMeta(props: MetaProps) {
  return makeJsx("meta", props, []);
}
