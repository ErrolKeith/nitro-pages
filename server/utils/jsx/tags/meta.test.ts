import { it, expect } from "vitest";
import { makeMeta, MetaProps } from "./meta";

it("can make a mata tag", () => {
  const metaTestConfig: MetaProps = {
    name: "keywords",
    content: "asdfasdfs",
  };
  const metaTest = makeMeta(metaTestConfig);
  expect(metaTest).toMatchSnapshot();
});
