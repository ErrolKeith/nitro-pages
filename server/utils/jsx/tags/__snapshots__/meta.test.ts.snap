// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`can make a mata tag 1`] = `
<meta
  content="asdfasdfs"
  name="keywords"
/>
`;
